code/datums/craft/menu.dm

```diff
@@ -51,9 +51,11 @@
 	data["cur_item"] = null
 
 	if(CR)
+		var/datum/asset/spritesheet/sheet = get_asset_datum(/datum/asset/spritesheet/craft)
+		var/list/result_item = get_craft_item(CR.result)
```

Comment:
Consider a check for this list existing here?

Reply:
> Consider a check for this list existing here?

It will exist with 100% probability, so no checks are needed.